File: web/back/models/group.py
from __future__ import annotations

from uuid import uuid4

from models.entity import AgeRanges
from models.utils import CleverBaseModel as BaseModel


class GroupModel(BaseModel):
    group_id: str | None = uuid4()
    organization_id: str
    name: str
    age_range: AgeRanges

File: web/front/kids-journal/src/types/group.ts
export enum AgeRanges {
  ZERO_THREE = "0-3",
  TWO_SIX = "3-6",
  SIX_PLUS = "6-9",
}

export type group = {
  group_id: string;
  organization_id: string;
  name: string;
  age_range: AgeRanges;
};

export type groups = [group];

export const GROUP: group = {
    group_id: "",
    organization_id: "",
    name: "",
    age_range: AgeRanges.ZERO_THREE,
};

export const GROUPS: groups = [GROUP];

